src/styles/blocks/inner-block.scss

```diff
@@ -0,0 +1,48 @@
+.inner-block {
+  z-index: 1;
+
+  width: 90%;
+  height: 90%;
+
+  background-color: $bg-color;
+  @include b-radius;
+
+  &::before {
+    @include position-absolute-half-block;
+    z-index: 2;
+
+    width: 15px;
+    height: 15px;
+
+    background-color: $primary-dark-color;
+    @include b-radius;
+
+    transform: translate(-50%, -50%);
+
+    content: "";
+  }
+
+  &__arrow {
+    @include position-absolute-half-block;
+    transform: translateX(-50%);
+  }
+}
+
+.arrow {
+  z-index: 1;
+
+  border-radius: 2px;
+
+  transform-origin: 0;
+
+  @each $type, $width, $height, $animation, $color in $arrows-params {
+    &--#{$type} {
+      width: $width;
+      height: $height;
```

Comment:
A `rectangle` mixin? Can be widely reused.

Reply:
fixed